lib/post_message_definition.yml

```diff
@@ -26,19 +26,21 @@ post_messages:
         payload:
           <<: *user_session_properties
           initial_step:
-            - search
-            - selectMember
-            - enterCreds
-            - mfa
-            - connected
-            - loginError
-            - disclosure
+            type:
```

Comment:
So just to make sure I understand `type` here. The value of `type` can be an actual type, like `string`, but it can also be a hash or an array?

Reply:
@sambev correct

Reply:
That's an existing feature though, what's new is the explicit "type" property.